timely/src/progress/frontier.rs

```diff
@@ -317,6 +317,13 @@ impl<T: PartialOrder+Ord+Clone> MutableAntichain<T> {
         self.frontier().less_equal(time)
     }
 
+    /// Clones the vector of updates.
+    /// Only used for debugging purposes.
```

Comment:
Can you say a bit more about what this is used for in the comments? If this lands, I'll need to support it, and it would help to understand why it is here and under which circumstances it could go away.

Reply:
Removed the function as it's no longer needed.